.circleci/config.yml

```diff
@@ -8,43 +8,44 @@ parameters:
     type: boolean
     description: Whether the pipeline was triggered through the CircleCi API (https://circleci.com/docs/api/v2/?shell#trigger-a-new-pipeline).
     default: false
-  production:
-    type: boolean
-    description: Whether to deploy to production (integreat.app) or not (webnext.integreat-app.de).
-    default: false
+  build_config:
```

Comment:
Based on my comment in the webpack config and package.json I think it makes sense to add here a boolean parameter `environment_production` or something named similar to set whether a development or production build should be made.

Reply:
That way it is possible to manually deploy:

- a malteser development build 
- integreat development build 
- integreat production build using the test cms.

Reply:
Honestly, I don't think we want to do this, especially I'm really sure we don't want to deploy a debug build/test cms build to production.

Reply:
The only use case I'm seeing is deploying a production build to webnext for debugging purposes like the ie11 error.

Reply:
So imo we should have the following possibilites:
- Build config: Integreat, Deploy to: integreat.app, Production build: true
- Build config: Integreat test cms, Deploy to: webnext, Production build: true/false
- Build config: Malte, Deploy to malte.xyz, Production build: true
(At some point: Build config: Malte test cms, Deploy to malte webnext, Production build: true/false)
And I honestly don't now if we need the possibility to deploy a production test cms build at all.

Reply:
As discussed in slack, won't do.